frontend/src/components/widgets/NumberInput/NumberInput.scss

```diff
@@ -56,6 +57,12 @@
           transition: none;
         }
 
+        &:active,
+        &:focus {
+          outline: none;
+          border: none;
+        }
```

Comment:
Hmmm... this means when I press Tab to focus the button it never gets visually selected. So that's no good :(

Please fix